src/resources/js/tickets.js

```diff
@@ -240,11 +240,14 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 						$( '#ticket_start_date' ).val( start_date );
 						$( '#ticket_end_date' ).val( end_date );
 
+						var $start_meridian = $( document.getElementById( 'ticket_start_meridian' ) );
+						var $end_meridian = $( document.getElementById( 'ticket_end_meridian' ) );
```

Comment:
I'll make this change because it _is_ part of the WP standards, though I'm not a big fan of comma separated vars.

I subscribe to [Ben Alman's opinions](http://benalman.com/news/2012/05/multiple-var-statements-javascript/) on the matter. The points he brings up _plus_ the number of lines that get considered "changed" in a diff any time you want to add/remove a declaration just muddies the waters for code reviews.

This is probably something that needs some discussion at some point.